zerver/lib/events.py

```diff
@@ -991,6 +998,16 @@ def apply_event(
             user_status.pop(user_id_str, None)
 
         state["user_status"] = user_status
+    elif event["type"] == "stream_user_group_access":
+        if event["op"] == "create":
+            state["stream_user_group_access_data"].append(event["stream_user_group_access_object"])
+            state["stream_user_group_access_data"].sort(key=lambda obj: obj["id"])
```

Comment:
I did not understand why this should be sorted. Do we use this fact somewhere in the frontend?

Reply:
No this is not used in frontend, at all. Just added because all other companions like `realm_users`, `realm_bots` which are keyed using id are sorted always in state. May be I'll remove this as I get feedback for this from others too.